src/domains/functions/function_services.tf

```diff
@@ -137,8 +132,8 @@ locals {
       APIM_SUBSCRIPTION_KEY                  = data.azurerm_key_vault_secret.fn_services_io_service_key.value
       PAGOPA_ECOMMERCE_API_KEY               = data.azurerm_key_vault_secret.fn_services_pagopa_ecommerce_api_key.value
       BETA_USERS                             = data.azurerm_key_vault_secret.fn_services_beta_users.value
-      SENDING_FUNC_API_KEY                   = data.azurerm_key_vault_secret.sending_func_key.value
-      SENDING_FUNC_API_URL                   = data.azurerm_key_vault_secret.sending_func_url.value
+      SENDING_FUNC_API_KEY                   = data.azurerm_key_vault_secret.rc_func_key.value
+      SENDING_FUNC_API_URL                   = "https://io-p-itn-com-rc-func-01.azurewebsites.net"
```

Comment:
Cannot be this value referred from a data?

Reply:
[a9ffb7c](https://github.com/pagopa/io-infra/pull/1590/commits/a9ffb7c54e7da0cb1f0c48bffae0d55c362ed42b)